dashboard/test/factories.rb

```diff
@@ -42,7 +42,7 @@
 
   factory :user do
     birthday Date.new(1991, 3, 14)
-    sequence(:email) {|n| "testuser#{n}@example.com.xx"}
+    sequence(:email) {|n| "testuser#{n}_#{SecureRandom.hex}@example.com.xx"}
```

Comment:
@joshlory it's not guaranteed to be unique in the case of rerunning this new seed task on a given environment. It will still be unique for unit tests that are run in a transaction.